### PR TITLE
Added some government websites

### PR DESCRIPTION
**Pull request by asubodh**

### CODE REVIEW

**Review by thenishantsapkota — DISMISSED** [review]

**Comment by thenishantsapkota** [comment]
Thank you @asubodh for your contributions.

**Comment by thenishantsapkota** [comment]
Also one thing, let's try to include 2-3 data in a single PR.

**Comment by asubodh** [comment]
> Also one thing, let's try to include 2-3 data in a single PR.

Sure 🚀

File: data/websites.json [hunk 1]
```diff
@@ -8,7 +8,22 @@
...
+    },
+    {
+      "name": "Government of Nepal Ministry Of Youth and Sports",
+      "url": "https://moys.gov.np/",
+      "description": "Government of Nepal Ministry Of Youth and Sports Official Site"
+    },
+    {
+      "name": "Ministry of Health and Population",
+      "url": "https://mohp.gov.np/",
+      "description": "Government of Nepal Ministry of Health and Population"
```
Comment (thenishantsapkota) [inline data/websites.json:26]:
```suggestion
      "description": "Government of Nepal Ministry of Health and Population's official website"
```

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

**Review by thenishantsapkota — APPROVED** [review]

**Comment by thenishantsapkota** [comment]
@asubodh Please resolve the merge conflicts!

**Review by TheLearneer — APPROVED** [review]
LGTM

**Pull request merged**